ntoskrnl/mm/ARM3/expool.c

```diff
@@ -1704,8 +1739,20 @@ ExpFindAndRemoveTagBigPages(IN PVOID Va,
     // the lock and return the tag that was located
     //
     InterlockedIncrement((PLONG)&Entry->Va);
```

Comment:
We're holding the `ExpLargePoolTableLock`, is the atomic operation still needed?

Reply:
No. And setting bit 0 like this is bizarre.

Reply:
Fixed.